src/Test/Desktop/OpenRiaServices.Common.DomainServices.Test/Cities/CityDomainService.cs

```diff
@@ -293,14 +293,22 @@ public string Echo(string msg)
         }
 
         [Invoke]
-        public async Task<string> EchoWithDelayAsync(string msg, TimeSpan delay)
+        public async Task<string> EchoWithDelay(string msg, TimeSpan delay)
         {
             // This method is used to test cancellation of invoke operations
             // Since the method might return to soon otherwise we add a delay
-            await Task.Delay(delay, ServiceContext.CancellationToken);
+            await Delay(delay);
             return Echo(msg);
         }
 
+        [Invoke]
+        public Task Delay(TimeSpan delay)
```

Comment:
If you remove this method you dont need to update code generation tests

Reply:
done